File: Go/uri_1037.go
package main

import "fmt"

func main() {
	var n float64
	fmt.Scan(&n)
	if n > 25 && n <= 50 {
		fmt.Printf("Intervalo (25,50]\n")
	} else if n >= 0 && n <= 25 {
		fmt.Printf("Intervalo [0,25]\n")
	} else if n >= 75 && n <= 100 {
		fmt.Printf("Intervalo (75,100]\n")
	} else {
		fmt.Printf("Fora de intervalo\n")
	}
}

File: Python/uri_1037.py
x = float(input());

if 0 <= x <= 25.0000:
    print("Intervalo [0,25]")
elif 25.0000 < x <= 50.0000:
    print("Intervalo (25,50]")
elif 50.0000 < x <= 75.0000:
    print("Intervalo (50,75]")
elif 75.0000 < x <= 100.0000:
    print("Intervalo (75,100]")
else:
    print("Fora de intervalo")
